<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTracking</class>
 <widget class="QWidget" name="FormTracking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Tracking</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblJudul">
     <property name="styleSheet">
      <string notr="true">font-size:18px; font-weight:bold; color:#2b5fab;</string>
     </property>
     <property name="text">
      <string>Form Input Tracking</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>

   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblTrackingID">
       <property name="text">
        <string>ID Tracking</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtTrackingID"/>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="lblPengirimanID">
       <property name="text">
        <string>ID Pengiriman</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtPengirimanID"/>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="lblKurirID">
       <property name="text">
        <string>ID Kurir</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtKurirID"/>
     </item>

     <item row="3" column="0">
      <widget class="QLabel" name="lblStatus">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cmbStatus">
       <item>
        <property name="text">
         <string>dikemas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>dikirim</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>selesai</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gagal</string>
        </property>
       </item>
      </widget>
     </item>

     <item row="4" column="0">
      <widget class="QLabel" name="lblLokasi">
       <property name="text">
        <string>Lokasi</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtLokasi"/>
     </item>

     <item row="5" column="0">
      <widget class="QLabel" name="lblWaktuUpdate">
       <property name="text">
        <string>Waktu Update</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateTimeEdit" name="dateTimeUpdate">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm:ss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnSimpan">
       <property name="text">
        <string>Simpan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUbah">
       <property name="text">
        <string>Ubah</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCari">
       <property name="text">
        <string>Cari</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHapus">
       <property name="text">
        <string>Hapus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBersih">
       <property name="text">
        <string>Bersih</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <item>
    <widget class="QTableWidget" name="tableTracking">
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>ID Tracking</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Pengiriman</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Kurir</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Lokasi</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Waktu Update</string>
      </property>
     </column>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
